Sources/TSCBasic/misc.swift

```diff
@@ -347,8 +347,14 @@ extension SystemError: CustomStringConvertible {
         switch self {
         case .chdir(let errno, let path):
             return "chdir error: \(strerror(errno)): \(path)"
-        case .close(let errno):
-            return "close error: \(strerror(errno))"
+        case .close(let err):
+            let errorMessage: String
+            if err == -1 { // if the return code is -1, we need to  consult the global `errno`
```

Comment:
comment formatting nit
```suggestion
            if err == -1 { // if the return code is -1, we need to consult the global `errno`
```